test/e2e/docs/running-tests.md

```diff
@@ -2,112 +2,168 @@
--B [branch]  - Run Jetpack tests on given Jetpack branch via https://jurassic.ninja
--s  - Screensizes in a comma-separated list (defaults to mobile,desktop)
--g  - Execute general tests in the specs/ directory
--j  - Execute Jetpack tests in the specs-jetpack-calypso/ directory (desktop and mobile)
--W  - Execute WooCommerce tests in the specs-woocommerce/ directory (desktop and mobile)
--C  - Execute tests tagged with @canary
--J  - Execute Jetpack connect tests tagged with @canary
--H [host]  - Specify an alternate host for Jetpack tests
--w - Only execute signup tests on Windows/IE11, not compatible with -g flag
--z  - Only execute canary tests on Windows/IE11, not compatible with -g flag
--y  - Only execute canary tests on Safari 10 on Mac, not compatible with -g flag
--l [config]  - Execute the tests via Sauce Labs with the given configuration
--c  - Exit with status code 0 regardless of test results
--m [browsers]  - Execute the multi-browser visual-diff tests with the given list of browsers via grunt.  Specify browsers in comma-separated list or 'all'
--i  - Execute i18n NUX screenshot tests, not compatible with -g flag
--I  - Execute tests in specs-i18n/ directory
--x  - Execute the tests from the context of xvfb-run
--u [baseUrl]  - Override the calypsoBaseURL config
--h  - This help listing
+```
+./run.sh -g -s <mode1>,<mode2>
 ```
 
-## To run headlessly
+### Headless
 
 By default the tests start their own Selenium server in the background, which in turn launches a Chrome browser on your desktop where you can watch the tests execute. This can be a bit of a headache if you're trying to do other work while the tests are running, as the browser may occasionally steal focus back (although that's mostly been resolved).
 
-The easiest way to run "headlessly" without a visible window is to add the `-x` flag when running `run.sh` or using the `HEADLESS=1` environment variable which will run Chrome with the --headless flag.
+The easiest way to run "headlessly" without a visible window is to add the `-x` flag when running `run.sh`:
 
-1. `./run.sh -g -x`
+```
+./run.sh -g -x
+```
 
-or
+or using the `HEADLESS=1` environment variable which will run Chrome with the `--headless` flag:
+
+```
+export HEADLESS=1
+./node_modules/.bin/mocha <path_to_e2e_spec>
+```
+
+### Other options
+
+The `run.sh` script takes a number of parameters that can be mixed-and-matched.
+
+For the list of current supported flags, use `run.sh -h`.
+
+
+### CircleCI docker image
+
+WIP.
+
+
+## CircleCI
+
+Broadly speaking, `Automattic/wp-calypso` runs two types of test suites:
+
+* canary (limited set)
+* full (full set)
+
+Each suite is also scheduled to execute every _x_ hours at certain times throughout the day.
+
+### Canary
+
+As of 2021-01-25, canary tests are triggered _automatically_ on all PRs and merges.
+
+### Full suite
+
+The following labels provide ability to trigger e2e tests for specific feature(s):
```

Comment:
I am not sure if we should put this here or not. I guess it doesn't matter, but these won't work for external users

Reply:
If it does not have to strictly remain internal, I would prefer to leave here - would prefer to see GitHub documentation be the ongoing, living documentation of e2e tests and this would align with the goal.

With that said, I am open to moving this to a consolidated FG page as well.

Reply:
I'm fine with leaving it.